Calling/ClientApp/src/components/ChatThread.tsx

```diff
@@ -322,6 +343,30 @@ export default (props: ChatThreadProps): JSX.Element => {
             <Chat
               styles={chatStyle}
               items={messagesWithAttached.map((message: any, index: number) => {
+                if (isUploadedFileMessage(message.content)) {
+                  const messageContent = JSON.parse(message.content);
+                  if (!messageContent) {
+                    return {};
+                  }
+
+                  return {
+                    key: index,
+                    contentPosition: message.mine ? 'end' : 'start',
+                    message: (
+                      <Card aria-roledescription="image card">
```

Comment:
Nice to have: Show the display name of the user who sent the file. I'm working on storing ACS user IDs in Storage, which I'm guessing would be required to achieve this, so this can definitely go on the backlog.

Reply:
sounds good, i'll create a follow up ticket